File: tags/3.3.32/Plugins.NET/Hello.CS/bin/Hello.cs
using FarManager;
public class HelloCS : BasePlugin
{
	void item_OnOpen(object sender, OpenPluginMenuItemEventArgs e)
	{
		Far.Msg("Hello, world from c#", "Far.NET");
	}
	void sayHello(string s)
	{
		Far.Msg("C#: Hello, " + s);
	}
	override public void Connect()
	{
		Far.RegisterPluginsMenuItem("Hello c#", item_OnOpen);
		Far.RegisterPrefix("hellocs", sayHello);
	}
}

File: tags/3.3.32/Plugins.NET/Hello.VB.NET/bin/Hello.vb
Imports FarManager
Public Class HelloVB
Inherits BasePlugin
	Sub item_OnOpen(ByVal sender As Object, ByVal e As OpenPluginMenuItemEventArgs)
		Far.Msg("Hello, world from vb.net", "Far.NET")
	End Sub

	Sub sayHello(ByVal s As String)
		Far.Msg("VB.NET: Hello, " + s)
	End Sub

	Public Overloads Overrides Sub Connect()
		Far.RegisterPluginsMenuItem("Hello vbnet", AddressOf item_OnOpen)
		Far.RegisterPrefix("hellovb", AddressOf sayHello)
	End Sub
End Class
